README.md

```diff
@@ -370,12 +443,10 @@ The superuser action MUST be denoted `*`. This is the maximum ability and may be
 
 This is useful in several cases, for example:
 
-1. When delegating all resources, like in a [`my` scheme](#4.2-all-owned-resources)
+1. When delegating all resources, like in a [`my` scheme](#42-owned-resources)
 2. To grant the maximum ability when the current ability semantics may be extended later
 
-## 4.2 All Owned Resources
-
-### 4.2.1. `my` and `as` Schemes
+## 4.2 Owned Resources
 
 The `my` URI scheme represents ownership over a resource — typically by parenthood — at decision-time (i.e. the validator's "now"). Resources that are created after the UCAN was created MUST be included. This higher-order scheme describes delegating some or all ambient authority to another DID.
```

Comment:
> The `my` URI scheme represents ownership over a resource — typically by parenthood — at decision-time (i.e. the validator's "now").

"typically by parenthood" ? What are the other ways to represent ownership?

> Resources that are created after the UCAN was created MUST be included.

Whats does this mean? Its just something like, if you are creating a new file you need to define that file in the URI ?

Reply:
>  Resources that are created after the UCAN was created MUST be included.

My understanding is that at the time of UCAN creation `my:*` may have included certain set of capabilities. If in the future `my:*` has changed to include greater set of capabilities old UCAN with `my:*` would still include those capabilities, or in other words  you  could derive another ucan with specific capability that did not existed when proof with `my:*` was issued.

Reply:
So its more "Resources that are created after the UCAN is issued are implicitly included." does this make sense ?

Reply:
I like that way to phrase it. Maybe re-adding the "MUST" to keep the spec terminology: "Resources that are created after the UCAN was issued MUST be implicitly included."?